frontend/src/services/gqlAuth.js

```diff
@@ -7,6 +8,16 @@ module.exports = {
     return !isTokenExpired(token);
   },
 
+  getRole() {
+    var token = localStorage.getItem('jwt_user_token');
```

Comment:
@briaguya: This is more of a question than a suggestion. Why do we get the role off the token? What is the `decode()` function about? Would like to understand more about the overall front end security considerations around jwt tokens.

Also, was curious about support for `localStorage` and found this. Sounds very edge case but good to be aware:
> Note: Starting with iOS 5.1, Safari Mobile stores localStorage data in the cache folder, which is subject to occasional clean up, at the behest of the OS, typically if space is short. Safari Mobile's Private Browsing mode also prevents writing to localStorage entirely.
https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage